lib/git-manager.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
     */
    getPull: function(repo, number) {
       debug("Getting pull %s", number);
-      return github.pullRequests.get(params({ number }, repo))
+      return rateLimit(github.pullRequests.get)(params({ number }, repo))
```

Comment:
There's a little piece of me wonders if we should iterate over the namespaces in `github` and the functions in each namespace, and apply the `rateLimit` to each of them. I suspect it might be more complex than it's worth, thought. What do you think?

Reply:
Thought about it, but I don't see us using very many more over time and it seems more likely that we'd mess that up, than just to forget one here.

Reply:
Fair.